tf-psa-crypto/drivers/builtin/src/sha256.c

```diff
@@ -414,7 +414,7 @@ static size_t mbedtls_internal_sha256_process_many_a64_crypto(
     return processed;
 }
 
-int mbedtls_internal_sha256_process_a64_crypto(mbedtls_sha256_context *ctx,
+static int mbedtls_internal_sha256_process_a64_crypto(mbedtls_sha256_context *ctx,
                                                const unsigned char data[SHA256_BLOCK_SIZE])
```

Comment:
Should the continuation lines be lined up to just after the opening parenthesis? (multiple places)